### ISSUE TITLE
Puzzling or problematic statements pulled out of Pathway Commons by biopax_api

### ISSUE THREAD

**Issue by johnbachman** [sorgerlab/indra#30]
I'm going to use this issue to keep an ongoing record of suspicious statements pulled out of Pathway Commons. Many (most, all?) of these may be either errors or deliberate curation shortcuts put in the database itself, but I'll list these as an issue in case they are appearing because we are doing something incorrectly in the api.

GTP as a kinase:
```
Phosphorylation(GTP, MAPK10, PhosphorylationThreonine, 221),
 Phosphorylation(GTP, MAPK10, PhosphorylationTyrosine, 223),
 Phosphorylation(GTP, MAPK8, PhosphorylationThreonine, 183),
 Phosphorylation(GTP, MAPK8, PhosphorylationTyrosine, 185),
 Phosphorylation(GTP, MAPK9, PhosphorylationThreonine, 183),
 Phosphorylation(GTP, MAPK9, PhosphorylationTyrosine, 185),
```

H, K, and NRAS as kinases (more understandable):
```
Phosphorylation(HRAS, BRAF, PhosphorylationSerine, 578),
 Phosphorylation(HRAS, BRAF, PhosphorylationThreonine, 373),
 Phosphorylation(HRAS, RAF1, PhosphorylationSerine, 338),
 Phosphorylation(HRAS, RAF1, PhosphorylationSerine, 621),
 Phosphorylation(HRAS, RAF1, PhosphorylationThreonine, 268),
 Phosphorylation(HRAS, RAF1, PhosphorylationThreonine, 269),
Phosphorylation(KRAS, BRAF, PhosphorylationSerine, 578),
 Phosphorylation(KRAS, BRAF, PhosphorylationThreonine, 373),
 Phosphorylation(KRAS, RAF1, PhosphorylationSerine, 621),
 Phosphorylation(KRAS, RAF1, PhosphorylationThreonine, 268),
 Phosphorylation(KRAS, RAF1, PhosphorylationThreonine, 269),
Phosphorylation(NRAS, BRAF, PhosphorylationSerine, 578),
 Phosphorylation(NRAS, BRAF, PhosphorylationThreonine, 373),
 Phosphorylation(NRAS, RAF1, PhosphorylationSerine, 621),
 Phosphorylation(NRAS, RAF1, PhosphorylationThreonine, 268),
 Phosphorylation(NRAS, RAF1, PhosphorylationThreonine, 269),
```

IL2 (ligand) as a kinase (perhaps this is meant as indirect?):
```
Phosphorylation(IL2, MAPK1, PhosphorylationThreonine, 182),
 Phosphorylation(IL2, MAPK1, PhosphorylationTyrosine, 184),
 Phosphorylation(IL2, MAPK3, PhosphorylationThreonine, 201),
 Phosphorylation(IL2, MAPK3, PhosphorylationTyrosine, 203),
```

KSR1 as a kinase:
```
Phosphorylation(KSR1, MAPK1, PhosphorylationThreonine, 182),
 Phosphorylation(KSR1, MAPK1, PhosphorylationTyrosine, 184),
 Phosphorylation(KSR1, MAPK3, PhosphorylationThreonine, 201),
 Phosphorylation(KSR1, MAPK3, PhosphorylationTyrosine, 203),
```

14-3-3 proteins as kinases. Could it be these are involved in some kind of reaction describing negative regulation that's not getting parsed correctly?
```
Phosphorylation(YWHAB, MAPK1, PhosphorylationThreonine, 182),
 Phosphorylation(YWHAB, MAPK1, PhosphorylationTyrosine, 184),
 Phosphorylation(YWHAB, MAPK3, PhosphorylationThreonine, 201),
 Phosphorylation(YWHAB, MAPK3, PhosphorylationTyrosine, 203),
 Phosphorylation(YWHAE, MAPK1, PhosphorylationThreonine, 182),
 Phosphorylation(YWHAE, MAPK1, PhosphorylationTyrosine, 184),
...
Phosphorylation(p14_3_3_family, MAPK1, PhosphorylationThreonine, 182),
 Phosphorylation(p14_3_3_family, MAPK1, PhosphorylationTyrosine, 184),
 Phosphorylation(p14_3_3_family, MAPK3, PhosphorylationThreonine, 201),
 Phosphorylation(p14_3_3_family, MAPK3, PhosphorylationTyrosine, 203),
```

Inconsistency in sequence numbering for MAPK1. Also note that these two statements are not only inconsistent with each other, but also with the phosphorylation statements shown above for MAPK1, which lists the residues 182 and 184:
```
ActivityModification(MAPK1, ['PhosphorylationThreonine', 'PhosphorylationTyrosine'], [183, 185], increases, Activity),
 ActivityModification(MAPK1, ['PhosphorylationThreonine', 'PhosphorylationTyrosine'], [185, 187], increases, Activity),
```

RAS mediates dephosphorylation of RAF1?
```
Dephosphorylation(HRAS, RAF1, PhosphorylationSerine, 621),
 Dephosphorylation(KRAS, RAF1, PhosphorylationSerine, 621),
 Dephosphorylation(NRAS, RAF1, PhosphorylationSerine, 621),
 Dephosphorylation(RAS_family, RAF1, PhosphorylationSerine, 621),
```

**Comment by bgyori** [comment]
Some of these are indeed problems with the pattern searches in our processor. Some of these are fixed in: 545c3a26b110aa1cb43c9a116a2be25ac6def3c7

**Comment by bgyori** [comment]
These are fixed by 827b587f8b32ec72cab06b2f7ed07ebd4c3264de, if we find more similar ones later, we can create new issues.
